### PR TITLE
Enable pin info page

### PR DESCRIPTION
**Pull request by frederikrothenberger**
This PR adds the pin info page to the pin registration flow.

### CODE REVIEW

File: src/frontend/src/flows/register/index.ts [hunk 1]
```diff
@@ -76,6 +77,11 @@ export const registerFlow = async <T>({
   }
   const result_ = await (async () => {
```
Comment (peterpeterparker) [inline src/frontend/src/flows/register/index.ts:78]: it's me or the function as no return type defined?

Comment (frederikrothenberger) [reply src/frontend/src/flows/register/index.ts:78]: Yes, although the result is later destructured on lines 133 to 147 (after the `canceled` case has been handled).

**Review by peterpeterparker — APPROVED** [review]
Rubber stamp

File: src/frontend/src/flows/register/index.ts (hunk 1 above)
Comment (peterpeterparker) [reply src/frontend/src/flows/register/index.ts:78]: Ok

**Pull request merged**